agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/statsbeat/Instrumentations.java

```diff
@@ -69,30 +71,72 @@ class Instrumentations {
     INSTRUMENTATION_MAP.put("io.opentelemetry.lettuce-5.0", 30);
     INSTRUMENTATION_MAP.put("io.opentelemetry.lettuce-5.1", 31);
     INSTRUMENTATION_MAP.put("io.opentelemetry.spring-rabbit-1.0", 32);
-    INSTRUMENTATION_MAP.put("io.opentelemetry.jaxrs-client-2.0", 33);
-    INSTRUMENTATION_MAP.put("io.opentelemetry.jaxrs-client-2.0-resteasy-3.0", 36);
+    // index 33 retired in 3.3.0 GA (was jaxrs-2.0-client)
+    // index 34 retired in 3.3.0 GA (was log4j-1.2)
+    // index 35 retired in 3.3.0 GA (was log4j-2.0)
+    // index 36 retired in 3.3.0 GA (was jaxrs-client-2.0-resteasy-3.0)
+    // index 37 retired in 3.3.0 GA (was logback-1.0)
+    // TODO (trask) start capturing this
     INSTRUMENTATION_MAP.put("io.opentelemetry.micrometer-1.0", 38);
     INSTRUMENTATION_MAP.put("io.opentelemetry.mongo-3.1", 39); // mongo 4.0 is covered in 3.1
     INSTRUMENTATION_MAP.put("io.opentelemetry.grizzly-2.0", 40);
     INSTRUMENTATION_MAP.put("io.opentelemetry.quartz-2.0", 41);
-    INSTRUMENTATION_MAP.put("io.opentelemetry.apache-camel-2.20", 42);
+    INSTRUMENTATION_MAP.put("io.opentelemetry.apache-camel-2.20", 42); // no usage yet
     INSTRUMENTATION_MAP.put("io.opentelemetry.netty-4.0", 43);
     INSTRUMENTATION_MAP.put("io.opentelemetry.netty-4.1", 44);
     INSTRUMENTATION_MAP.put("io.opentelemetry.okhttp-3.0", 45);
-    INSTRUMENTATION_MAP.put("io.opentelemetry.opentelemetry-annotations-1.0", 46);
+    INSTRUMENTATION_MAP.put("io.opentelemetry.opentelemetry-extension-annotations-1.0", 46);
```

Comment:
@heyams can you make this rename in `javaInstrumentationBitMap3_2_3()`, and add the new mappings below?